Sources/Controllers/Map/Toolbars/OAWeatherToolbar.mm

```diff
@@ -98,8 +130,13 @@ - (void)commonInit
     
     self.timeSliderView.stepsAmountWithoutDrawMark = 145.0;
     [self.timeSliderView clearTouchEventsUpInsideUpOutside];
+    
+    _currentDate = [NSDate now];
+    _selectedDate = [NSDate now];
+    _animationState = EOAWeatherToolbarAnimationStateIdle;
+    
     [self.timeSliderView removeTarget:self action:NULL forControlEvents:UIControlEventTouchUpInside | UIControlEventTouchUpOutside];
-    [self.timeSliderView addTarget:self action:@selector(timeChanged:) forControlEvents:UIControlEventTouchUpInside | UIControlEventTouchUpOutside];
+    [self.timeSliderView addTarget:self action:@selector(timeChanged:) forControlEvents:UIControlEventTouchUpInside | UIControlEventTouchUpOutside | UIControlEventValueChanged];
 
```

Comment:
Was it tested on a real device? It seems to me that it's not a coincidence that UIControlEventValueChanged wasn't used here. It might affect performance